### PR TITLE
LPS-153858

### PR DESCRIPTION
**Pull request by holatuwol**
https://issues.liferay.com/browse/LPS-153858

I believe the main goal of the bug report was because the user wanted to configure the interval in minutes, but Liferay only allows configuration by hour. We can't change the meaning of the configuration (since people may have already customized it using config files), but we can change its type.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
@holatuwol 

We have to talk into the team about this LPS.
So for this reason, some time will be required.

sorry for the inconvenience.

File: modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/configuration/RedirectConfiguration.java [hunk 1]
```diff
@@ -53,6 +53,6 @@ public interface RedirectConfiguration {
 		description = "check-redirect-not-found-entries-interval-help",
 		name = "check-redirect-not-found-entries-interval", required = false
 	)
-	public int checkRedirectNotFoundEntriesInterval();
+	public float checkRedirectNotFoundEntriesInterval();
```
Comment (AliciaGarciaGarcia) [inline modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/configuration/RedirectConfiguration.java:56]: Hi @holatuwol 
We think that the use of a float to let the user introduce not integer numbers is not a good idea and it's no other use on the system like this.
The configuration help says:
´Set the time interval, in hours, between 404 URL checks. All 404 URLs stored in the database that exceed both the configured maximum age and maximum number will be removed.`

We do not understand the need of the check of the redirections for their removal on periods of time so shorts. But if this is needed we think that id better to use int and minutes as another configurations.

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/configuration/RedirectConfiguration.java (hunk 1 above)
Comment (holatuwol) [reply modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/configuration/RedirectConfiguration.java:56]: Understood, I'll go ahead and close the LPS as won't fix.

**Pull request closed without merging**